src/main/java/com/untamedears/humbug/Humbug.java

```diff
@@ -1531,6 +1531,73 @@ public void onPlayerBucketEmptyEvent(PlayerBucketEmptyEvent e) {
       }
     }
   }
+  
+  @BahHumbug(opt="drop_ores_as_ore", def="true")
+  @EventHandler(priority = EventPriority.HIGHEST, ignoreCancelled = true)
+  public void orebreak(BlockBreakEvent e) {
+	  if(!config_.get("drop_ores_as_ore").getBool()) {
+		  return;
+	  }
+	  if (isOre(e.getBlock().getType())) {
+		  e.setCancelled(true);
```

Comment:
Don't cancel the event if you still want your pretty graphs. Setting the block to air should be good enough.